recipes/_common.rb

```diff
@@ -21,22 +21,37 @@
 include_recipe 'chef-sugar::default'
 require 'chef/sugar/core_extensions'
 
-# Create the user
-include_recipe 'omnibus::_user'
```

Comment:
I'm confused on why we have to duplicate the `group` and `user` resources here and `include 'omnibus::_user'` later in the file. Why wouldn't we just leave this existing `include 'omnibus::_user'` here at the top of the file?